guides/deployment/gigalixir.md

```diff
@@ -96,7 +96,19 @@ $ echo 'node_version=12.16.3' > phoenix_static_buildpack.config
 If you are using Phoenix v1.6, there is also a `mix assets.deploy` alias in your `mix.exs` that you want to invoke post compilation:
 
 ```console
-$ echo 'hook_post_compile="mix assets.deploy && rm -f _build/esbuild"' >> elixir_buildpack.config
+$ echo 'hook_post_compile="mix assets.deploy && rm -f _build/esbuild"' >> phoenix_static_buildpack.config
```

Comment:
I think we simply don't want to do this line at all. :)

Reply:
Agreed =)